src/test/java/seedu/address/model/ModelManagerTest.java

```diff
@@ -26,31 +25,31 @@
     public void constructor() {
         assertEquals(new UserPrefs(), modelManager.getUserPrefs());
         assertEquals(new GuiSettings(), modelManager.getGuiSettings());
-        assertEquals(new AddressBook(), new AddressBook(modelManager.getAddressBook()));
+        assertEquals(new GradPad(), new GradPad(modelManager.getGradPad()));
     }
 
     @Test
     public void setUserPrefs_nullUserPrefs_throwsNullPointerException() {
-        assertThrows(NullPointerException.class, () -> modelManager.setUserPrefs(null));
+        Assertions.assertThrows(NullPointerException.class, () -> modelManager.setUserPrefs(null));
     }
 
     @Test
     public void setUserPrefs_validUserPrefs_copiesUserPrefs() {
         UserPrefs userPrefs = new UserPrefs();
-        userPrefs.setAddressBookFilePath(Paths.get("address/book/file/path"));
+        userPrefs.setGradPadFilePath(Paths.get("address/book/file/path"));
```

Comment:
Maybe we can rename the dummy path since we're no longer building addressbook 😁   Same goes for the change on line 46 below

```suggestion
        userPrefs.setGradPadFilePath(Paths.get("gradpad/file/path"));
```